core-bundle/src/Resources/contao/dca/tl_content.php

```diff
@@ -1206,8 +1207,10 @@ public function showJsLibraryHint($dc)
 			return;
 		}
 
+		$security = System::getContainer()->get('security.helper');
+
 		// Return if the user cannot access the layout module (see #6190)
-		if (!$this->User->hasAccess('themes', 'modules') || !$this->User->hasAccess('layout', 'themes'))
+		if (!$security->isGranted(ContaoCorePermissions::USER_CAN_ACCESS_MODULE, 'themes') || !$security->isGranted(ContaoCorePermissions::USER_CAN_ACCESS_LAYOUTS))
```

Comment:
Shouldn't this be `USER_CAN_ACCESS_THEME`?

Reply:
No 😅